README.md

```diff
@@ -50,6 +50,38 @@ Android) only, try:
 ./scripts/build-glean-metadata fenix
 ```
 
+For the search service _only_ (see below), the Glean Dictionary also indexes
+legacy Firefox telemetry metadata. To build this index (not necessary for most
+testing), you need to run a seperate script:
+
+```bash
+./scripts/build-legacy-metadata
+```
+
+## Search Service
+
+The Glean Dictionary also includes a search service which enables searching
+through active metrics. Under the hood, this service is implemented with
+[netlify functions]. For example:
+
+https://dictionary.telemetry.mozilla.org/.netlify/functions/metrics_search_burnham?search=techno
```

Comment:
something we might consider in a follow up is to simplify the api path [with redirects](https://www.jamesqquick.com/blog/how-to-redirect-netlify-functions-to-a-simpler-path).

Reply:
Definitely. 👍 Would you be interested in taking that on?